Packs/Dig/pack_metadata.json

```diff
@@ -1,12 +1,13 @@
 {
-    "name": "Dig",
+    "name": "Dig (Deprecated)",
     "description": "Run Dig automation to get 'A' and 'PTR' records.",
```

Comment:
Please add deprecated also to the pack description

Reply:
Fixed.